src/index.js

```diff
@@ -204,22 +204,30 @@ export const status = async (cluster, cid, { local, signal } = {}) => {
     signal
   })
 
-  let peerMap = data.peer_map
-  if (peerMap) {
-    peerMap = Object.fromEntries(
-      Object.entries(peerMap).map(([k, v]) => [
-        k,
-        {
-          peerName: v.peername,
-          status: v.status,
-          timestamp: new Date(v.timestamp),
-          error: v.error
-        }
-      ])
-    )
-  }
+  return toStausResponse(data)
+}
 
-  return { cid: data.cid['/'], name: data.name, peerMap }
+/**
+ * @param {API.Config} cluster
```

Comment:
Should we add the note in the PR post about this being heavy operation and that Filters should be used?